dev/reloadable-server/src/main/resources/play/reference-overrides.conf

```diff
@@ -1,3 +1,8 @@
 # Overriding Akka's defaults so that services are assigned different tcp ports.
 akka.remote.artery.canonical.port = 0
 akka.remote.artery.canonical.hostname = "127.0.0.1"
+
+# Overriding Akka's defaults for Remote Classic so that services are assigned different tcp ports.
```

Comment:
What are the implications of having both definitions here? Should we instead have something like we do for Akka HTTP and Netty in Play?

https://www.playframework.com/documentation/2.7.x/NettyServer#Usage

[`PlayNettyServer` plugin](https://github.com/playframework/playframework/blob/ab6697f494a000ba8d19768e9d04eed3bb8cdbf7/dev-mode/sbt-plugin/src/main/scala/play/sbt/Play.scala#L96-L108) basically add the correct dependency, which in Lagom's case should then bring a different `reference-overrides.conf` with settings for either Artery (default like Akka HTTP) or classic remoting (like NettyServer).

Reply:
Classic remote also requires Netty as a dependency:


> [info] [WARN] [06/28/2019 16:08:20.786] [a-impl-internal-dev-mode-akka.actor.internal-dispatcher-2] [CoordinatedShutdown(akka://a-impl-internal-dev-mode)] Task [shutdown-application-dev-mode] failed in phase [service-stop]: Unexpected exception[IllegalStateException: Classic remoting is enabled but Netty is not on the classpath, it must be added explicitly. See https://doc.akka.io/docs/akka/2.6/remoting.html]

From https://travis-ci.com/lagom/lagom/jobs/211851640#L1867.